tags.go

```diff
@@ -137,7 +137,7 @@ func alignTags(fields []*dst.Field) {
 			lenUsed := 0
 
 			if ok {
-				tagComponents = append(tagComponents, fmt.Sprintf("%s:\"%s\"", key, value))
+				tagComponents = append(tagComponents, fmt.Sprintf("%s:%#v", key, value))
```

Comment:
```suggestion
				tagComponents = append(tagComponents, fmt.Sprintf("%s:%q", key, value))
```

What about this?

Reply:
`%q` and `%#v` is the same for strings

`%#v` - https://github.com/golang/go/blob/cb4eee693c382bea4222f20837e26501d40ed892/src/fmt/print.go#L492

`%q` - https://github.com/golang/go/blob/cb4eee693c382bea4222f20837e26501d40ed892/src/fmt/print.go#L503